specs/beacon-chain.md

```diff
@@ -51,18 +51,20 @@ The primary source of load on the beacon chain are "attestations". Attestations
 * The `BASE_REWARD_QUOTIENT` constant is the per-slot interest rate assuming all validators are participating, assuming total deposits of 1 ETH. It corresponds to ~3.88% annual interest assuming 10 million participating ETH.
 * At most `1/MAX_VALIDATOR_CHURN_QUOTIENT` of the validators can change during each dynasty.
 
-**Status codes**
-
-| Status code | Value |
-| - | :-: |
-| `PENDING_LOG_IN` | `0` |
-| `LOGGED_IN` | `1` |
-| `PENDING_EXIT` | `2` |
-| `PENDING_WITHDRAW` | `3` |
-| `WITHDRAWN` | `4` |
-| `PENALIZED` | `128` |
-| `ENTRY` | `1` |
-| `EXIT` | `2` |
+**Codes, flags, types**
```

Comment:
I think it will be clear if you split it into three tables.

Reply:
Will do!